doc/Cluster_Setup.md

```diff
@@ -7,8 +7,7 @@ Also see https://github.com/Seagate/cortx-hare/blob/dev/README.md
 
```

Comment:
Please take a look at LINE 4. We should be the `main` branch. So the link should be:
```
https://github.com/Seagate/cortx-hare/blob/main/README.md
```